databuilder/publisher/neo4j_csv_publisher.py

```diff
@@ -16,6 +17,16 @@
 from databuilder.publisher.neo4j_preprocessor import NoopRelationPreprocessor
 
 
+# Setting field_size_limit to solve the error below
+# _csv.Error: field larger than field limit (131072)
+maxInt = sys.maxsize
+while True:
+    try:
+        csv.field_size_limit(maxInt)
+        break
+    except OverflowError:
+        maxInt = int(maxInt / 10)
```

Comment:
do you encounter overflow error during the testing?

Reply:
And did you test the long nested type table from zippy?

Reply:
No, I didn't test that. But Stackflow says it might.